diode-server/gen/diode/v1/reconcilerpb/reconciler.pb.go

```diff
@@ -77,6 +77,58 @@ func (State) EnumDescriptor() ([]byte, []int) {
 	return file_diode_v1_reconciler_proto_rawDescGZIP(), []int{0}
 }
 
+type ActionIngestionLogRequest_Action int32
+
+const (
+	ActionIngestionLogRequest_ACTION_UNSPECIFIED ActionIngestionLogRequest_Action = 0
+	ActionIngestionLogRequest_ACTION_DIFF        ActionIngestionLogRequest_Action = 1
+	ActionIngestionLogRequest_ACTION_APPLY       ActionIngestionLogRequest_Action = 2
+	ActionIngestionLogRequest_ACTION_REJECT      ActionIngestionLogRequest_Action = 3
+)
```

Comment:
feels a bit java-verbose here I think in the generation due to nesting+prefixing perhaps

Reply:
Indeed, we can also extract this enum definition outside of `ActionIngestionLogRequest` message and name it `ActionType` or something like that, then it would generate just `ActionType_*` constants

Reply:
yeah this is about as terse as we're likely to get from it I think.  I used `UNSPECIFIED_ACTION` here because it complained about UNSPECIFIED being a part of the State enum.